pom.xml

```diff
@@ -81,11 +92,27 @@
             <version>2.22.2</version>
             <scope>test</scope>
         </dependency>
+
+        <dependency>
+            <groupId>org.jboss.arquillian.junit</groupId>
+            <artifactId>arquillian-junit-container</artifactId>
+            <version>1.4.0.Final</version>
```

Comment:
cannot this version be taken from imported arquillian bom?

Reply:
unfortunately no... at least what I know...http://arquillian.org/modules/core-platform/#artifacts